### DISCUSSION TITLE
main.js failing to load on netlify

### DISCUSSION THREAD

**Discussion by darkins** [fylgja/11ty-starter-template#6] [Q&A]
Hi,

For some reason **main.js** is not loading when i push to netlify. 
I can see it in my **_site/js** directory, but not finding it when i go to my browser and type: "whatever".netlify.app/js/main.js
It works absolutely fine locally without any issue. 

Is this perhaps to do with 'use strict' or maybe cross origin incompatibility with netlify?

Any help greatly appreciated.

thanks

**Comment by GrimLink** [comment]
@darkins I'm not sure; I'd have to look at the Netlify logs to see what's wrong.

You could check if there is a an error in the logs with the build step.

However, because this starter template is also hosted on Netlify, it should work right away.

As you can see the logs for this starter template show the correct output.

![Screenshot 2023-01-29 at 12 40 44](https://user-images.githubusercontent.com/4387541/215342856-209f974d-8585-4c14-925e-bc2f61540d6c.jpg)

If there is something wrong on the JS build step with Rollup, you should see it in the logs.

It is also unrelated to 'use strict' or cross-origin incompatibility.
